catalog_publish.java

```diff
@@ -148,6 +159,30 @@ private void processCatalog(Path platformYaml) {
 
     }
 
+    private void publishCatalogMembers(byte[] parentPlatform, String repository) throws IOException
+    {
+        ExtensionCatalog catalog = CatalogMapperHelper.deserialize(new ByteArrayInputStream(parentPlatform),
+                 io.quarkus.registry.catalog.ExtensionCatalogImpl.Builder.class);
+        Map<String, Object> platformRelease = (Map<String, Object>) catalog.getMetadata().get("platform-release");
+        List<String> members = (List<String>) platformRelease.get("members");
+        for (String member : members) {
+            // Skip this catalog
+            if (!member.equals(catalog.getId())) {
+                ArtifactCoords memberCoords = ArtifactCoords.fromString(member);
+                String groupId = memberCoords.getGroupId();
+                String artifactId = memberCoords.getArtifactId();
+                String version = memberCoords.getVersion();
+                String classifier = version;
+                byte[] jsonPlatform = readCatalog(repository, groupId, artifactId, version, classifier);
+                // Publish
+                log.infof("Publishing %s:%s:%s", memberCoords.getGroupId(), memberCoords.getArtifactId(), memberCoords.getVersion());
+                String platformKey = memberCoords.getGroupId() + ":" + memberCoords.getArtifactId();
```

Comment:
Would `client/platforms` and the Maven equivalent return members as their own platforms?

Reply:
@aloubyansky yes, I was a bit hesitant about returning that, but it seems to make sense in the end. WDYT?

Reply:
> @aloubyansky yes, I was a bit hesitant about returning that, but it seems to make sense in the end. WDYT?

Actually, `/client/platforms/all` does that. Because of  https://github.com/quarkusio/registry.quarkus.io/pull/92/commits/eb2e2d69d594329790a86cf17fefcea98f612491 I'm only listing the default platform (which TBH I am thinking to drop it)

Reply:
No, members aren't platforms.